### PR TITLE
populate: use self while calling add_pub_interface

### PR DESCRIPTION
**Pull request by theanalyst**
Since it is a method in the class we need to call it with self.
otherwise we'll end up with a NameError

Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

### CODE REVIEW

**Comment by theanalyst** [comment]
Missed this in #95 review, we really need a linter to check these :)

File: srv/modules/runners/populate.py [hunk 1]
```diff
@@ -569,14 +569,14 @@ def igw_members(self):
...
 
     def ganesha_members(self):
         """
         Create a file for ganesha hosts.
         """
         minion_dir = "{}/role-ganesha/stack/default/{}/minions".format(self.root_dir, self.cluster)
-        add_pub_interface(minion_dir)
+        self.add_pub_interface(minion_dir)
 
     def add_pub_interface(self, minion_dir):
```
Comment (jan--f) [inline srv/modules/runners/populate.py]: maybe make ```add_pub_interface``` "private"?

Comment (theanalyst) [inline srv/modules/runners/populate.py]: ie. `_add_pub_interface`?

Comment (theanalyst) [inline srv/modules/runners/populate.py]: @jan--f updated

**Comment by rjfd** [comment]
@theanalyst I agree we should have pylint running on travis, but before we activate such thing we have to cleanup all the python code according to the linter rules.
Can you open an issue to track the use of pylint?

**Comment by theanalyst** [comment]
@rjfd opened https://github.com/SUSE/DeepSea/issues/107

File: srv/modules/runners/populate.py (hunk 1 above)
Comment (jan--f) [inline srv/modules/runners/populate.py]: Is it is only used within the class even ```__add_pub_interface```. It won't be truly private but at least the intent is clear.

**Pull request merged**